portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -2360,6 +2515,91 @@ definition {
 		}
 	}
 
+	@description = "LPS-186875 Confirm that the tip and action creation options are displayed on the page"
+	@priority = 3
+	test CreationActionsAdminPageIsDisplayed {
+		task ("And goes to Creation Action tab") {
+			Button.click(button = "Creation Actions");
+		}
+
+		task ("Then the "New Creation Action" button is present in the page") {
+			CustomerPortal.assertButtonIsVisible(text = "New Creation Action");
+		}
+
+		task ("And the text “No actions were created. Start creating one action to interact with your data.” is displayed") {
+			AssertTextPresent(
+				locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT",
+				value1 = "No actions were created.");
+
+			AssertTextPresent(
+				locator1 = "ObjectCustomViews#VIEW_BUILDER_EMPTY_STATE_TEXT",
+				value1 = "Start creating an action to interact with your data.");
+		}
+	}
+
+	@description = "LPS-186875 Confirm that an action creation can be saved with a different translation, and using the Side Panel type"
+	@priority = 5
+	test HeadlesFieldIsNotMandatoryInSidePanelOnCreationAction {
```

Comment:
name of the test should be Headle**s**sFieldIsNotMandatoryInSidePanelOnCreationAction